index.js

```diff
@@ -299,10 +299,19 @@ const extractFeatures = async page => {
   }
 }
 
+let sharp = null
+const resizeCanvas = async (canvas, resX, resY) => {
+  if (!sharp) sharp = require("sharp")
+  const sharpCanvas = sharp(canvas)
+  return sharpCanvas.resize(resX, resY).toBuffer()
```

Comment:
Maybe here before resizing, we could check if current res is <= (resX, resY), and if so not resize.
Could avoid a resize operation, and would be better in terms of quality for pixel art for instance.

Reply:
Note: there's a dark-arts optimisation here, which would be to return the canvas width/height when running `captureCanvas()`, inside the page context. Would allow bypassing loading sharp if the image is small enough (as we need sharp to get width/height from a buffer).
Let's leave a todo comment, and we'll do this later™️